docs/migrating-1x-2x.md

```diff
@@ -0,0 +1,19 @@
+Migrating from AutoDispose 1.x to 2.x
+
+AutoDispose 2.x is built against RxJava 3.x and is binary-compatible with AutoDispose 1.x and RxJava 2.x. As such - it has a different package name and maven group ID.
```

Comment:
Binary _incompatible_ right?

Reply:
No, binary compatible. They can both be present in the same classpath with no conflicts.

Reply:
Ahh i see what you mean.